src/modules/wallet/utils/account.js

```diff
@@ -303,3 +304,48 @@ export const validate2ndPass = async (account, passphrase, error) => {
   }
   return messages;
 };
+
+/**
+ * Validate a derivation path
+ * TODO: Replace this function when @liskhq (lisk-sdk) exposes the one they use
```

Comment:
could you please GitHub issue in SDK repo and link the issue here?

Reply:
Do you mean that we shall wait to merge this until response from SDK?

https://github.com/LiskHQ/lisk-sdk/issues/7877

Reply:
No we dont need to wait, in line 310 TODO we can refer the above SDK issue and we can clean this up once SDK exposes the function